### DISCUSSION TITLE
Expose ports

### DISCUSSION THREAD

**Discussion by freire-nuno** [hassio-addons/addon-node-red#873] [Q&A] [answered]
I was trying to use a collection to interact with alexa (node-red-contrib-alexa-remote2)and it needs to access port 3456. How can I expose it? Is it possible via configuration?

**Comment by romufk** [comment]
Hi, I have same problem to expose UDP port.

No solution ?

**Comment by frenck** [comment, marked as answer]
The add-on runs on the host network... there is no such thing as "exposing" ports in that case.
